src/utils/renderSchema.ts

```diff
@@ -80,6 +80,10 @@ export const renderSchema: Walker = function*(schema, dereferencedSchema, level
       if (node.properties !== undefined) {
         const isConditionalCombiner = node.combiner === 'anyOf' || node.combiner === 'oneOf';
```

Comment:
add allOf here?

Reply:
Sounds good! Will make the change in a bit.